src/runtime/browser/util.inspect.polyfil.ts

```diff
@@ -377,19 +377,18 @@ function reduceToSingleString(output: string[], base: string, braces: string[]):
   return braces[0] + (base === "" ? "" : base + "\n") + "  " + output.join(",\n  ") + " " + braces[1];
 }
 
-function _extend(origin: object, add: object): object {
-  const typedOrigin = { ...origin } as { [key: string]: unknown };
+function _extend(origin: object, add: object): void {
+  const typedOrigin = origin as {[key: string]: unknown};
```

Comment:
It has to be exactly the same object. If it is cloned, only `typedOrigin` is extended, not `origin`.